modules/stochastic_tools/test/tests/ics/neural_net_ic/evaluate.i

```diff
@@ -0,0 +1,32 @@
+# [StochasticTools]
+# []
+#
+# [Samplers]
+#   [test]
+#     type = CartesianProduct
+#     linear_space_items = '0.25 1 10
+#                           0.25 1 10
+#                           0.25 1 10'
+#   []
+# []
+#
+# [VectorPostprocessors]
+#   [results]
+#     type = SurrogateTester
+#     model = surrogate
+#     sampler = test
+#     execute_on = final
+#   []
+# []
+
```

Comment:
remove commented out stuff. doe this produce anything without an executioner block?!

Reply:
This input file is meant to run the neural_net_ic.i (another input file in same directory). It compares the result of the nn_ic simulation to a gold exodus file, to confirm the neural net is evaluated correctly and applied to the ic variable.